bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -665,15 +655,15 @@ lnd/clm2/surfdata_map/surfdata_1x1_brazil_16pfts_Irrig_CMIP6_simyr2000_c171214.n
 
 <!-- Crop surface datasets -->
 <fsurdat hgrid="C384"  sim_year="2000" use_crop=".true." >
-lnd/clm2/surfdata_map/surfdata_C384_78pfts_CMIP6_simyr2000_c181018.nc</fsurdat>
+lnd/clm2/surfdata_map/surfdata_C384_78pfts_CMIP6_simyr2000_c200317.nc</fsurdat>
```

Comment:
Put these under the release-clm5.0.30 subdirectory as well, like the others. Looks like the filename isn't quite right either as it's missing "_hist_" before _78pfts.

Reply:
Done. Fixed file path to reflect Erik's comments.